sdk/lib/atl/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+
+add_library(atl_classes INTERFACE)
+
+target_include_directories(atl_classes INTERFACE ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Nice trick

Reply:
Actually, this should probably only be for C++

And yes, that's nice because you can move things around without worrying too much.